### DISCUSSION TITLE
Question - Is it possible to add bots to an online server if its all running on the same system? (no UUID)

### DISCUSSION THREAD

**Discussion by JeflolxSpursf** [PrismarineJS/mineflayer#2602] [Q&A]
Is it possible to add bots without a UUID to an online server in my case a paper server if its hosted on the same system?
 
Edit: Wait I could just use localhost....

**Comment by u9g** [comment]
what do you mean by bots without a uuid?

**Reply by JeflolxSpursf** [reply]
An offline / cracked account that isn't a paid login.

**Comment by ghost** [comment]
If u meant connecting a cracked account into a premium/online servers, the answer is no.
